src/content/self-hosted/administration/configuration.mdx

```diff
@@ -623,6 +483,137 @@ registry:
         size: 40Gi
 ```
 
+##### azure
+
+Use this to use Azure Storage to store your private images.
+
+```yaml
+registry:
+  storage:
+    provider:
+      azure:
+        enabled: true
+        storage:
+          container: "storage-container-name"
+          accountName: "service-account-name"
+```
+
+Then, create a secret like this:
```

Comment:
can we include a link to a doc on how to create this token, which privileges or IAM it needs, and that stuff?